templates/posts.html

```diff
@@ -5,18 +5,16 @@
   {% if loop.index0 % 3 == 0 %}
   <div class="row u-equal-height u-clearfix">
   {% endif %}
-    <div class="col-4 p-card">
+    <div class="col-4 p-card--resources">
       <header class="p-card__header--{{ post.groups[0].slug }}">
         <h5 class="p-muted-heading">{{ post.groups[0].name }}</h5>
```

Comment:
The padding around the headings are slightly off, because the line-height is different here than in /resources. I forgot to put an h tag inside header so in mine it's treated like a div. Anyway, Spencer preferred the padding in /resources so I think we should both use `<h4>` and change the `.p-card__header` padding to `0.875rem 1rem` to get the correct spacing.